src/sql/workbench/browser/modelComponents/radioCardGroup.component.html

```diff
@@ -1,30 +1,29 @@
-<div role="radiogroup" *ngIf="cards" class="card-group" style="flex-wrap:wrap" [style.height]="height"
-	[style.width]="width" [attr.aria-label]="ariaLabel" (keydown)="onKeyDown($event)">
-	<div #cardDiv role="radio" *ngFor="let card of cards" class="model-card" (click)="selectCard(card.id)"
-		[attr.aria-checked]="isCardSelected(card.id)" [tabIndex]="getTabIndex(card.id)" [style.width]="cardWidth"
-		[style.height]="cardHeight" (focus)="onCardFocus(card.id)" (blur)="onCardBlur(card.id)" style="flex:0 0 auto;">
-		<span class="selection-indicator-container">
-			<div *ngIf="isCardSelected(card.id)" class="selection-indicator"></div>
-		</span>
-		<div class="card-vertical-button">
-			<div *ngIf="card.icon" class="iconContainer">
-				<div [class]="getIconClass(card.id)" [style.width]="iconWidth" [style.height]="iconHeight"></div>
+<div [class]="orientation">
```

Comment:
Same here - couldn't you just add it to the existing class list on the radiogroup div?